src/RuntimeState.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef GENSET_WHISPERPOWER_DDC_RUNTIME_STATE_HPP
+#define GENSET_WHISPERPOWER_DDC_RUNTIME_STATE_HPP
+
+#include <base/Time.hpp>
+
+namespace genset_whisperpower_ddc {
+    struct RuntimeState {
+        base::Time time;
+        
+        base::Time total_runtime; // Total run time to be reset after maintenance
+        base::Time historical_runtime;
+    };
```

Comment:
I have to admit it's a bit of nitpicking but ...

"runtime" one word means "while running", at least when related to software. Here, it is "run time", that is how long it ran. So, please rename to RunTimeState, total_run_time and historical_run_time